accepted/0000-better-cobbler-tftp-sync.md

```diff
@@ -0,0 +1,77 @@
+- Feature Name: Better Cobbler TFTP Sync
+- Start Date: 2020-11-08
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+More optimized TFTP syncing for Cobbler proxies. Switching from push to pull.
+
+# Motivation
+[motivation]: #motivation
+
+For large scale installations Cobbler has proven to be a possible bottleneck. One of the problems is the Cobbler sync that, when using it in conjunction with proxies, also pushes the TFTP folder to the proxies. For installations with just one proxy and around 130 profiles the whole process can easily take up to 15 minutes. Not all of that is due to TFTP sync, but a good chunk of it and might force clients into a reboot, since information are not yet updated.
```

Comment:
```suggestion
For large scale installations Cobbler has proven to be a possible bottleneck. One of the problems is the Cobbler sync that, when using it in conjunction with proxies, also pushes the TFTP folder to the proxies. For installations with just one proxy and around 130 profiles the whole process can easily take up to 15 minutes. Not all of that is due to TFTP sync, but a good chunk of it is. Note also that clients might require unnecessary reboots in case they are booting during the sync, since TFTP information is not updated atomically.
```

Does the above wording keep the original meaning?

Reply:
Yes!
A real atomic update will not be possible in my eyes due to the fact that we would need to update the data at the very same moment. When having a few distros in SUMA or Cobbler we are talking about a kernel and initrd for each one, as well as other (smaller) files. But I am open for ideas.